### PR TITLE
Disable nightly local tree builds.

### PR DESCRIPTION
**Pull request by jgadling**
### Summary:
- **What:** `Disable nightly local tree builds now that ondemand tree builds is live, and remove unused group.yaml`
- **Ticket:** [sc150146](https://app.shortcut.com/genepi/story/150146)
- **Env:** `https://cron-frontend.dev.genepi.czi.technology`

### Demos:
I temporarily updated terraform code to create cron events for rdev and I only see contextual builds:
Screen Shot 2021-09-30 at 11 00 47 AM

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/aspen/cli/db.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def add_can_see(
     "--builds-template-file",
     type=str,
     required=False,
-    default="src/backend/aspen/workflows/nextstrain_run/builds_templates/group.yaml",
+    default="src/backend/aspen/workflows/nextstrain_run/builds_templates/group_plus_context.yaml",
```
Comment (danrlu) [inline src/backend/aspen/cli/db.py:201]: what is this used for? (purely out of curiousity)

**Comment by danrlu** [comment]
should we also remove
nextstrain_profile/aspen_auspice_config.json
nextstrain_profile/covidtrackerca.md?
or I can do a clean up of nextstrain_profile folder soon and do that alongside ;)

**Review by danrlu — APPROVED** [review]
All looks good. The other change is optional

File: src/backend/aspen/cli/db.py (hunk 1 above)
Comment (jgadling) [reply src/backend/aspen/cli/db.py:201]: it technically doesn't get used at all - it's the default value to use if a --builds-template-file argument isn't specified (and we always specify one.) But referring to a missing file just seemed like a bad idea 🤷‍♀️

**Review by lvreynoso — APPROVED** [review]

**Pull request merged**